src/StripeTests/Entities/BillingPortal/Sessions/ConfigurationTest.cs

```diff
@@ -0,0 +1,26 @@
+namespace StripeTests.BillingPortal
+{
+    using Newtonsoft.Json;
+    using Stripe;
+    using Stripe.BillingPortal;
+    using Xunit;
+
+    public class ConfigurationTest : BaseStripeTest
+    {
+        public ConfigurationTest(StripeMockFixture stripeMockFixture)
+            : base(stripeMockFixture)
+        {
+        }
+
+        [Fact]
+        public void Deserialize()
+        {
+            var json = GetResourceAsString("api_fixtures.billing_portal.configuration.json");
```

Comment:
Same as java, stripe-mock can return the fixture, so we should not be using a fixture here